libraries/AP_VideoTX/AP_VideoTX.cpp

```diff
@@ -61,7 +61,7 @@ const AP_Param::GroupInfo AP_VideoTX::var_info[] = {
     // @DisplayName: Video Transmitter Options
     // @Description: Video Transmitter Options. Pitmode puts the VTX in a low power state. Unlocked enables certain restricted frequencies and power levels. Do not enable the Unlocked option unless you have appropriate permissions in your jurisdiction to transmit at high power levels.
     // @User: Advanced
-    // @Bitmask: 0:Pitmode,1:Pitmode until armed,2:Pitmode when disarmed,3:Unlocked
+    // @Bitmask: 0:Pitmode,1:Pitmode until armed,2:Pitmode when disarmed,3:Unlocked,4:Pull the port down before sending a request
```

Comment:
I'm trying to think how we can word this so that it makes sense for users. Maybe label the option "Add leading zero byte" and then add to the documentation in description something like "Adds a leading zero to all requests to ensure that the line is low before the request is sent. Some VTX's are unable to correctly parse SmartAudio packets without this"

Reply:
Yeah fair